### PR TITLE
Added dymamic library symbols

### PR DESCRIPTION
**Pull request by gdanezis**
Apparently OpenSSL keeps side information about which functions to include in the dymanic library (the libcrypto.so file). I just added those.

### CODE REVIEW

File: util/mkdef.pl [hunk 1]
```diff
@@ -260,6 +260,9 @@
 $crypto.=" include/openssl/dsa.h" ; # unless $no_dsa;
 $crypto.=" include/openssl/dh.h" ; # unless $no_dh;
 $crypto.=" include/openssl/ec.h" ; # unless $no_ec;
+
```
Comment (benlaurie) [inline util/mkdef.pl]: Why the blank lines?

**Comment by benlaurie** [comment]
Other than the blank lines, this whole repo needs rebasing, libeay.num has moved on since it was cloned.

Otherwise, this looks fine to me.

**Comment by dfaranha** [comment]
Done! The library now builds as a shared object and tests/bench are correctly linked. Thanks!

**Comment by benlaurie** [comment]
@dfaranha looks like you haven't updated this repo...

**Comment by benlaurie** [comment]
Silly me, I was looking at the wrong branch.

**Comment by dfaranha** [comment]
Perhaps I should have committed on branch pairings and do a squashed version on this branch, I will try adjusting later.

**Pull request merged**